stdlib/3/multiprocessing/__init__.pyi

```diff
@@ -72,19 +72,25 @@ class Pool():
     def __exit__(self, exc_type, exc_val, exc_tb) -> None: ...
 
 class Process():
+    name: str = ...
+    daemon: bool = ...
+    pid: Optional[int] = ...
+    exitcode: Optional[int] = ...
+    authkey: bytes = ...
+    sentinel: int = ...
     # TODO: set type of group to None
     def __init__(self,
-                 group: Any = ...,
+                 group: Any = None,
                  target: Callable = ...,
-                 name: str = ...,
+                 name: Optional[str] = None,
                  args: Iterable[Any] = ...,
                  kwargs: Mapping[Any, Any] = ...,
-                 daemon: bool = ...) -> None: ...
+                 daemon: Optional[bool] = None) -> None: ...
```

Comment:
While you're at it, this argument is keyword-only (so there should be a `*` before daemon).